app.py

```diff
@@ -17,15 +17,23 @@
                         help="""The crn or course subject and number.
                                 If using the course subject and number, they must be enclosed in quotation marks.
                                 For instance: \"MATH 2214\" """)
+    parser.add_argument('-p', '--pushbullet', action='store_true', help='Send notification with pushbullet')
+    parser.add_argument('-d', '--desktop', action='store_true', help='Send alert via libnotify')
+    parser.add_argument('-f', '--config', help='Specify alternative config file')
```

Comment:
Could this be -c so it's consistent with the others?

Reply:
Easy enough to do; -f is just slightly more consistent with short flags for other tools I've used; I think I had an intended use for -c and got sidetracked elsewhere.